docs/source/codedoc.rst

```diff
@@ -106,6 +106,29 @@ Further reading:
 - `Python Standard Library: typing -- Support for type hints <https://docs.python.org/3/library/typing.html>`_
 - `PEP 483 -- The Theory of Type Hints <https://www.python.org/dev/peps/pep-0483/>`_
 
+
+Properties
+----------
+
+A method with a decoration ending in ``property`` will be included in the generated API documentation as an attribute rather than a method::
```

Comment:
It is clear from the examples, buy maybe we can be more catholic  than the Pope...  you can ignore it or do it as a drive by for another PR.

```suggestion
A method with a decoration ending in ``property`` (case-insensitive) will be included in the generated API documentation as an attribute rather than a method::
```

Reply:
The code doesn't do a full case-insensitive compare; it only accepts `property` and `Property`; not `proPERty` for example.

I tried to keep the docs simple, but maybe I oversimplified here. I'll add `Property` to the docs.

Reply:
The docs update is excellent. Thanks!